src/test/java/com/orgofarmsgroup/controller/UserControllerTest.java

```diff
@@ -24,7 +25,7 @@ class UserControllerTest {
     @DisplayName("users root api")
     void testUsers() throws Exception {
         mockMvc.perform(
-                get("/users")
+                get(AppConstants.API_URI.USERS_API_URI.USERS_ROOT)
         ).andDo(print()).andExpect(status().isOk());
```

Comment:
Can you verify the data received in the API response as well?

Reply:
sure